src/python/pants/goal/auxiliary_goal.py

```diff
@@ -43,7 +43,7 @@ class AuxiliaryGoal(ABC, GoalSubsystem):
     which must run "outside" of the usual engine processing to function.
     """
 
-    # Used by `pants.option.arg_splitter.ArgSplitter()` to optionally allow aliasing auxiliary goals.
+    # Used by the arg splitter to optionally allow aliasing auxiliary goals.
```

Comment:
Could have done these separately (or in a previous PR) but might as well clean it up here.